src/AppBundle/Resources/public/js/app.deck.js

```diff
@@ -913,6 +915,21 @@
             }
             return _.uniq(names).length >= 7;
         };
+        var validate_valyrian_steel = function() {
+            var i, n;
+            var names = [];
+            var attachments = deck.get_cards(null, {type_code: 'attachment'});
+            for (i = 0, n = attachments.length; i < n; i++) {
+                if (attachments[i].indeck > 1) {
```

Comment:
I don't think this rule (and the PHP equivalent) is as strict as it needs to be. 

Consider a deck that has 1x Ghost (the attachment) and 2x Ghost (the character). (swap out your favorite Direwolf). That should fail validation, because you have more than 1 copy of Ghost by title. Currently, it looks like it will succeed since only the number of copies of the attachment itself are being considered.

Reply:
good catch, i'll fix that tonight.

Reply:
@ystros i'm solicited further input, and i'm getting some push-back on your interpretation. Having some second thoughts on this as well. 

In the scenario you outlined, the deck would have only _one_ copy of the Ghost _attachment_ by title, whether there are other same-titled cards of different type may not be relevant.

Reply:
i stand corrected, your interpretation is right. from the official rules: 

Copy (of a card)
A copy of a card is defined by title: any other card that
shares the same title is considered a copy, regardless of
cardtype, text, artwork, or any other characteristic of
the card(s).

will fix. thanks again!